src/Illuminate/Foundation/Exceptions/Renderer/Listener.php

```diff
@@ -28,6 +30,10 @@ public function registerListeners(Dispatcher $events)
     {
         $events->listen(QueryExecuted::class, [$this, 'onQueryExecuted']);
 
+        $events->listen([JobProcessing::class, JobProcessed::class], function () {
```

Comment:
Should probably also listen for:

- `Illuminate\Queue\Events\JobFailed`
- `Illuminate\Queue\Events\JobReleasedAfterException`

Reply:
Ignition didn't; so prefer to stick with what's used at the moment by all our users.